### PR TITLE
fix: support pre-aggregate in standalone

### LINKED ISSUE #1560: Create PreAggrTables in standalone mode
**Bug Description**
Only in cluster mode, it will create pre-aggr tables. We should do it also in standalone mode.

### PR DESCRIPTION
**Pull request by nautaa**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
fix #1560

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

File: src/catalog/base.cc [hunk 1]
```diff
@@ -44,7 +44,6 @@ ProcedureInfoImpl::ProcedureInfoImpl(const ::openmldb::api::ProcedureInfo& proce
 
     for (const auto& op : procedure.options()) {
         options_[op.name()] = op.value().value();
-        DLOG(INFO) << "[procedure option] " << op.name() << " : " << options_[op.name()];
```
Comment (nautaa) [inline src/catalog/base.cc:47]: in standalone mode, this will generate periodic logs on the command line.

**Review by zhanghaohit — APPROVED** [review]
LGTM

File: src/catalog/base.cc (hunk 1 above)
Comment (zhanghaohit) [reply src/catalog/base.cc:47]: ok to remove.

Actually it will only print when compiling with `Debug` mode.

**Comment by zhanghaohit** [comment]
[long time window - compute engine](https://trello.com/c/XMMF5TXl/6-long-time-window-compute-engine)

**Review by dl239 — APPROVED** [review]

**Pull request merged**